### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by GimignanoF** [josdejong/jsoneditor#893]
Please add Dark Mode to the new web app :)

**Comment by josdejong** [comment]
:smile: that could be nice, thanks for your suggestion Jimi. It's not on the highest prio though, I first want to focus on functionality.

**Comment by GimignanoF** [comment]
Yeah, of course, is better that the site actually works rather than being dark 😅 But glad you considered this ;)

**Comment by josdejong** [comment]
So many plans, so little time :) We can keep this issue open.
